frontend/src/components/Forms/AdvancedContactSearchForm.js

```diff
@@ -32,12 +32,12 @@ export class AdvancedContactSearchForm extends Component {
   };
 
   handleReset = () => {
+    this.setState({ contactType: "PER" });
     this.props.reset();
     this.props.handleSearch({}, true);
-    this.setState({ contactType: "PER" });
   };
 
-  handleContactTypeChange = (chars, value) => {
+  handleContactTypeChange = (value) => {
```

Comment:
what was this change for? Does this function still work correctly?

Reply:
if I remember right, the chars wasn't being used in the function

Reply:
good catch